torchvision/transforms.py

```diff
@@ -298,6 +342,16 @@ def __init__(self, size, padding=0):
             self.size = size
         self.padding = padding
 
+    def get_params(self, img):
```

Comment:
Same comment here about standalone functions

Reply:
I'm still a bit unsure about what's the best way to expose the transformation parameters.
What do you think about making this a `staticmethod`, so that it can be called without instantiating the class?